generator/config/expression/first.yaml

```diff
@@ -0,0 +1,13 @@
+# $schema: ../schema.json
+name: $first
+link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/first/'
+type:
+    - resolvesToAny
+encode: single
+description: |
+    Returns the result of an expression for the first document in an array.
+arguments:
+    -
+        name: expression
+        type:
+            - resolvesToArray
```

Comment:
```suggestion
            - resolvesToArray
tests:
    -
        name: 'Use in $addFields Stage'
        # Note: no link as this is not an official example
        pipeline:
            -
                $addFields:
                    firstItem:
                        $first: '$items'
```

Reply:
Thanks for the test. I removed the note as the lack of `link` will the marked that this is not an example from the documentation.

Reply:
I wasn't sure whether the generator requires a link, so I wanted to pointed out in the suggestion.